File: TAXCALCUS.js
console.log("Welcome to the US sales tax calculator!");
console.log("STATES");
console.log("1: Florida");
console.log("2: Georgia");
console.log("3: Tennessee");
console.log("4: South Carolina");
console.log("5: North Carolina");
console.log("6: Alabama");
console.log("7: Mississippi");

var state = parseInt(prompt("Please enter your state number: "));
var county = 0;

if (state == 1){
    console.log("You have selected: Florida.");
    county = parseInt(prompt("Select your county: "));
} else if (state == 2){
    console.log("You have selected: Georgia.");
    county = parseInt(prompt("Select your county: "));
} else if (state == 3){
    console.log("You have selected: Tennessee.");
    county = parseInt(prompt("Select your county: "));
} else if (state == 4){
    console.log("You have selected: South Carolina.");
    county = parseInt(prompt("Select your county: "));
} else if (state == 5){
    console.log("You have selected: North Carolina.");
    county = parseInt(prompt("Select your county: "));
} else if (state == 6){
    console.log("You have selected: Alabama.");
    county = parseInt(prompt("Select your county: "));
} else if (state == 7){
    console.log("You have selected: Mississippi.");
    county = parseInt(prompt("Select your county: "));
}

console.log("State: " + state);
console.log("County: " + county);

File: TAXCALCUS.py
print("Welcome to the US sales tax calculator!")
print("COUNTIES")
print("1: Florida")
print("2: Georgia")
print("3: Tennesee")
print("4: South Carolina")
print("5: North Carolina")
print("6: Alabama")
print("7: Mississippi")
tax = 0
state = 0
state = int(input("Please enter your state number: "))
if state == 1:
    print("You have selected: Florida.")
    flcounty = 0
    flcounty = int(input("Select your county. "))
elif state == 2:
    print("You have selected: Georgia.")
    gacounty = 0
    gacounty = int(input("Select your county. "))
elif state == 3:
    print("You have selected: Tennessee.")
    tncounty = 0
    tncounty = int(input("Select your county. "))
elif state == 4:
    print("You have selected: South Carolina.")
    sccounty = 0
    sccounty = int(input("Select your county. "))
elif state == 5:
    print("You have selected: North Carolina.")
    nccounty = 0
    nccounty = int(input("Select your county. "))
elif state == 6:
    print("You have selected: Alabama.")
    alcounty = 0
    alcounty = int(input("Select your county. "))
elif state == 7:
    print("You have selected: Mississippi.")
    mscounty = 0
    mscounty = int(input("Select your county. "))